### PR TITLE
Drawing Rectangle on Each Image Slice with standard mean deviation.

### PR DESCRIPTION
**Pull request by niks2060**
Drawing Rectangle on Each Image Slice with standard mean deviation.

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -466,7 +466,7 @@ papaya.Container.CreateSideNevigation = function (containerHTML) {
     containerHTML.append('<div class="side-navpanel" style="display:none">Cene Controls <hr/><button id="reverseCene" title="Reverse Cene"><span class="fas fa-backward fa-2x"></span></button><div><input class="input-range" orient="vertical" type="range" step="1" value="10" min="1" max="100"> <span class="range-value"></span></div><button id="repeatCene" style="background-color:green;" title="Repeat Cene"><span class="fas fa-refresh fa-2x"></span></button></div>');
     var rulerLineSize = "<div>Line Size : <select id='" + PAPAYA_RULER_LINE_WIDTH + "'><option value='1' selected=selected>1</option><option value='2'>2</option><option value='3'>3</option><option value='4'>4</option><option value='5'>5</option></select></div>";
     var rulerFontSize = "<div>Font Size : <select id='" + PAPAYA_RULER_FONT_SIZE + "'><option value='14' selected=selected>14</option><option value='16'>16</option><option value='18'>18</option><option value='20'>20</option><option value='25'>25</option></select></div>";
-    var rulerLengthSize = "<div>Length Size : <select id='" + PAPAYA_RULER_LENGTH_UNIT + "'><option value='mm' selected=selected>Milimeters</option>" +
```
Comment (rii-mango) [inline src/js/main.js:469]: Millimeters

**Review by rii-mango — APPROVED** [review]
If a rectangle's corners overlap, and then you try to drag one to increase the rectangle size, it will move both corners and it stays the same zero size.

File: src/js/main.js (hunk 1 above)
Comment (niks2060) [reply src/js/main.js:469]: That is default. You can change it from left popup tools configuration

**Comment by niks2060** [comment]
> If a rectangle's corners overlap, and then you try to drag one to increase the rectangle size, it will move both corners and it stays the same zero size.

Yes will resolved that too. this happened when you select or drag on both side and both points overlaps each other.

File: src/js/main.js (hunk 1 above)
Comment (rii-mango) [reply src/js/main.js:469]: In the UI it's spelled as milimeters with only one L

Comment (niks2060) [reply src/js/main.js:469]: @rii-mango  resolve this in new PR.

**Pull request merged**